app/src/main/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -27,24 +27,26 @@ class PopularMoviesViewModel @Inject constructor(
     private val _isLoaderVisible = MutableLiveData<Boolean>()
 
     init {
-        getPopularMovie()
+        uploadPopularMovie()
     }
 
     fun navigateToMovieDetailsFragment(movieId: Int) {
         popularMoviesFragmentNavigator.navigateToMovieDetailsFragment(movieId)
     }
 
-    private fun getPopularMovie() = viewModelScope.launch {
-        _isLoaderVisible.value = false
-        getPopularMoviesUseCase().fold(
-            onSuccess = { moviesModel ->
-                _isLoaderVisible.value = true
-                _popularMovies.value = moviesModel.toUi()
-            },
-            onFailure = {
-                _isLoaderVisible.value = true
-                popularMoviesFragmentNavigator.errorSnackBar("Something went wrong")
-            }
-        )
+    private fun uploadPopularMovie() {
```

Comment:
"upload" suggests executing some soft of HTTP POST here ;)

Reply:
I changed `upload` to `load`